distributed/tests/test_client.py

```diff
@@ -4130,6 +4130,21 @@ async def test_persist_workers_annotate(e, s, a, b, c):
     assert s.loose_restrictions == {total2.key} | {v.key for v in L2}
 
 
+@nodebug  # test timing is fragile
```

Comment:
What exactly is timing sensitive? is there anything one can do to make this more robust? We do have a lot of flaky tests and some additional information might be helpful to debug this if it ever becomes unstable

Reply:
I just copied it from the test above :)
It originate from https://github.com/dask/distributed/pull/1534, @pitrou do you remember why this was added?

Reply:
Not sure, but notice the `allow_other_workers` setting was different, so perhaps the `L1` keys ending up on `a` depended on execution being fast enough?

Reply:
> Not sure, but notice the `allow_other_workers` setting was different, so perhaps the `L1` keys ending up on `a` depended on execution being fast enough?

That make sense. It that case, we don't need it for my test, will remove it.